pkg/converters/ingress/defaults.go

```diff
@@ -96,6 +96,7 @@ func createDefaults() map[string]string {
 		types.GlobalModsecurityTimeoutIdle:       "30s",
 		types.GlobalModsecurityTimeoutProcessing: "1s",
 		types.GlobalModsecurityTimeoutServer:     "5s",
+		types.GlobalModsecurityUseCoraza:         "",
```

Comment:
Empty string is the zero value for the string type, so this whole line can be removed.